### PR TITLE
Workflow actions to schedule scrapping of PDF data and calculating metrics 

### PR DESCRIPTION
**Pull request by rscnt**
This PR adds three new  commands and one workflow to automate the following tasks:

- [Downloading covid19data](https://github.com/rscnt/etl-pipeline#downloading-covid19india-data)
- [Calculating metrics based on historical data](https://github.com/rscnt/etl-pipeline#calculating-metrics-based-on-historical-data)
- and [Extracting the "the ward-wise breakdown of positive cases"](https://github.com/rscnt/etl-pipeline#scrapping-bmc-covid-19-response-war-room-dashboard) from the "stop coronavirus in Mumbai key updates trends file"

Why did many files change?

- `black`, ran the black formatter and it executed some a lot of changes.
- Added new scripts using the code from notebooks or other more "manual"  python tasks like pdf scrapping to create commands

What is the result?

We know have a new [workflow](https://github.com/rscnt/etl-pipeline/actions/runs/252169812) to execute the previously mentioned tasks daily or at will (by using workflow_dispatch).

A couple of things to do are:

- Push result csv files to a `gh-pages` branch as it's done in the [covid19india repo ](https://github.com/covid19india/api/tree/gh-pages), github assumes that file will host static files and the service applies some special rules to it.
- How to update these commands? We can see that for example the notebook and scripts to calculate metrics has been updated and we need to find an easy way to maintain these commands.
- Should we also store the PDF files for each day? We can upload them as [artifacts](https://github.com/actions/upload-artifact) for each job.

### CODE REVIEW

File: data_pipeline/calculate_metrics.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "--output-path",
+    default="output/metrics.csv",
+    help="The path of the metrics result csv file",
+)
+@click.option(
+    "--city-name",
+    default="Mumbai",
+    help="The name of the city to extract data from the input file",
+)
+def calculate_city_stats_metrics(input_path, output_path, city_name):
```
Comment (saurabhmj) [inline data_pipeline/calculate_metrics.py:26]: This function would needs to be updated. I'll do that once this code is merged

File: data_pipeline/extract_history_command.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    help="A list of state and districts to extract data from.",
+    show_default=True,
+)
+@click.option(
+    "--output-file",
+    default="city_stats.csv",
+    help="The path of the new csv containing the cities stats from the v4 history json.",
+    show_default=True,
+)
+def extract_history_command(history_json_url, states_and_districts, output_file):
```
Comment (saurabhmj) [inline data_pipeline/extract_history_command.py:33]: we'll need to rebase this function with the latest changes too.

**Comment by saurabhmj** [comment]
Thanks a lot for wrapping the code with parameters and bringing it into a workflow! Just need some rebasing. LGTM

**Comment by rscnt** [comment]
I'll wait for your PR and update the related commands.

**Comment by saurabhmj** [comment]
> I'll wait for your PR and update the related commands.

Cool. I'll merge my PR and you can pick up the code from there. Feel free to let me know if any help is required

**Pull request merged**